Generating pipelines...
6 pipelines have been generated.

Pipeline1: Preprocessing the signal file.
[ 3:00PM] 1.extract_CEL_files
Traceback (most recent call last):
  File "/home/xchen/chencode/Betsy/scripts/run_protocol.py", line 203, in <module>
    main()
  File "/home/xchen/chencode/Betsy/scripts/run_protocol.py", line 192, in main
    parameters,clean_up=args.clean_up)
  File "/home/xchen/chencode/Betsy/scripts/run_protocol.py", line 69, in run_protocol
    out_files = rule_engine.run_pipeline(pipeline, objects,clean_up=clean_up)
  File "/home/xchen/chencode/Betsy/Betsy/rule_engine.py", line 224, in run_pipeline
    analysis,objects,pipeline_sequence,clean_up=clean_up)
  File "/home/xchen/chencode/Betsy/Betsy/rule_engine.py", line 169, in run_module
    analysis.parameters, objects, pipeline_sequence)
  File "/home/xchen/chencode/Betsy/Betsy/modules/extract_CEL_files.py", line 12, in run
    directory = module_utils.unzip_if_zip(single_object.identifier)
  File "/home/xchen/chencode/Betsy/Betsy/module_utils.py", line 465, in unzip_if_zip
    extract_from_zip(input_name,directory)
  File "/home/xchen/chencode/Betsy/Betsy/module_utils.py", line 457, in extract_from_zip
    zip.extractall(path=outdir)
  File "/usr/local/lib/python2.7/zipfile.py", line 964, in extractall
    self.extract(zipinfo, path, pwd)
  File "/usr/local/lib/python2.7/zipfile.py", line 952, in extract
    return self._extract_member(member, path, pwd)
  File "/usr/local/lib/python2.7/zipfile.py", line 997, in _extract_member
    shutil.copyfileobj(source, target)
  File "/usr/local/lib/python2.7/shutil.py", line 49, in copyfileobj
    buf = fsrc.read(length)
  File "/usr/local/lib/python2.7/zipfile.py", line 583, in read
    data = self.read1(n - len(buf))
  File "/usr/local/lib/python2.7/zipfile.py", line 635, in read1
    max(n - len_readbuffer, self.MIN_READ_SIZE)
zlib.error: Error -3 while decompressing: invalid stored block lengths
